libs/cdk/src/lib/zone-configuration/zone-cfg.ts

```diff
@@ -25,30 +25,49 @@ const zoneSymbol = '__zone_symbol__';
 /**
  * https://angular.io/guide/zone#setting-up-zonejs
  **/
-function assertZoneConfig() {
+function assertZoneObjectPresentBeforeZoneJs() {
```

Comment:
```suggestion
function assertZoneObjectExistsBeforeZoneJs() {
```

be careful with this one please :D did not make sure that usages are refactored as well

Reply:
done